vhdbuilder/packer/collect-windows-logs.ps1

```diff
@@ -0,0 +1,47 @@
+$ProgressPreference="SilentlyContinue"
```

Comment:
Instead of relying on the file from aks-engine, I copied the file here because the file needs no frequent changes.

Reply:
Please create a folder `windows` under `vhdbuilder/scripts` and move this file to the new folder. Please also delete the old file under `vhdbuilder/scripts/`

Reply:
And please update this file with the latest content in `https://raw.githubusercontent.com/Azure/aks-engine/master/scripts/collect-windows-logs.ps1`

Reply:
Also update the path in vhdbuilder/packer/windows-vhd-builder.json